### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Foxhunter-de** [chrisvel/tududi#15] [Ideas]
Hello,
many thanks for Tududi! Although I've only been using it for a relatively short time, I really enjoy using it. I would like to use it in my national language (German). Is it planned to be able to work with different languages? I could help with the German translation.

Best regards!

**Comment by chrisvel** [comment]
Hello and thank you for your support. Add multilingual support for tududi is on the roadmap. I will share it the following days. I can definitely ping you when I start working on the German one. 

Cheers!

**Comment by franco-lamas** [comment]
Hello There

As a native Spanish speaker I believe I can assist in making Tududi more accessible to Spanish-speaking users. If you’re open to this idea, please let me know how I can start contributing or if there’s a specific process to follow.

**Reply by chrisvel** [reply]
Once i18n is in place, it should work with any language so I will definitely need some native speakers per language to verify auto translated stuff :stuck_out_tongue: 
Thanks!

**Comment by dpdp** [comment]
Ні! I want to help with the Ukrainian translation! I like your tududi))

**Reply by chrisvel** [reply]
Thank you, noted!

**Comment by chrisvel** [comment]
Hi @PCE-Consulting, @franco-lamas and @dpdp! 
Translations added (i18n) on v0.39 for German, Ukrainian and Spanish across others. 
They are not 100% complete but will be soon.
Feel free to check out and give feedback. 
Thanks!

**Comment by chrisvel** [comment]
Please post any other languages you would like to see available!
Thanks
